src/themes/dark/theme.css

```diff
@@ -238,6 +238,10 @@ html {
     background: #333;
 }
 
+.listItem:hover {
+    background: #252528;
```

Comment:
I think the `Dark` theme should be monochrome, but the buttons (`Sort`, `Filter`, etc.) are blue when hovered, so I dunno :man_shrugging:

Reply:
I know what you mean. But the menu that opens when you click on the three lines has a grey hover effect. I thought that mimicking that behavior would be fitting. Sort and filter are only small icons. Maybe thats why they are blue when hoverd?

Reply:
I mean, it could be `#252525` (equal channels).

Btw, there is `actionSheetMenuItem:hover` (it should go after `listItem:hover` to work properly):
https://github.com/jellyfin/jellyfin-web/blob/958e938f373b6239a0b46acbc774fe423d286c4d/src/themes/dark/theme.css#L175-L177
For `Dark` we could take `#242424` and remove `actionSheetMenuItem:hover` since `actionSheetMenuItem` is a `listItem`.
For other themes, the background color under the menu may be different, so we need that custom style.